hwdb/20-usb-vendor-model.hwdb

```diff
@@ -96,7 +96,7 @@ usb:v0200p0201*
  ID_MODEL_FROM_DATABASE=MA180 UMTS Modem
 
 usb:v0204*
- ID_VENDOR_FROM_DATABASE=Chipsbank Microelectronics Co., Ltd
+ ID_VENDOR_FROM_DATABASE=Chipsbank
```

Comment:
Dropping "co." and ", ltd" is reasonable, but the name of the company is actually "Chipsbank Microelectronics", so I think keeping "Microelectronics" would be preferable.